tests/Main.hs

```diff
@@ -788,6 +788,7 @@ runClashTest = defaultMain $ clashTestRoot
         , outputTest "T2510" def{hdlTargets=[VHDL], clashFlags=["-DNOINLINE=OPAQUE"]}
 #endif
         , outputTest "T2542" def{hdlTargets=[VHDL]}
+        , runTest "T2593" def{hdlSim=[]}
```

Comment:
```suggestion
        , runTest "T2593" def{hdlLoad=[], hdlSim=[]}
```
We don't need to "load" them in HDL tools either, it's enough that it compiles.

Reply:
Ah it occurred to me to check and I see what you did is a pattern that's often used in the test suite... so maybe I was just overthinking it and it should just be as the rest is. Feel free to disregard this suggestion.